### ISSUE TITLE
Animation a bit laggy

### ISSUE THREAD

**Issue by Sangrene** [bgryszko/react-native-circular-progress#150]
The circular animation is a bit laggy on iOS, is there any way to use native driver to smooth performances ?

Thanks

**Comment by markusl** [comment]
Which iOS version? Do you see the issue in the simulator or actual phones?

**Comment by Sangrene** [comment]
Tested on real devices (iPhone 6 & 8) using iOS 9.x & 11.x

**Comment by markusl** [comment]
@Sangrene or @RukaiJohn could you describe in which circumstances and example code this happened? I tested this with iPhone XS and didn't see any lag. 

You could also try with latest version as I updated the dependencies in the latest version.

**Comment by sepehr-laal** [comment]
I am here to report that animations are laggy on Android as well. On Pixel XL with Android 9.0.
I tried using the native driver but it entirely disables all animations.

**Comment by doug-stritt** [comment]
I'm also running into this issue on the iOS simulator running 12.1, react native 0.57.5, react 16.6.1 and RNSVG 9.2.4.

**Comment by RukaiJohn** [comment]
It's laggy in all phones I tried it on. It doesn't matter if it's modern or old,

**Comment by cyucelen** [comment]
Yes I am experiencing the lag too. Is there any way to overcome from this issue?

**Comment by deepslam** [comment]
The animation is lagging a lot! I used it on a real device and if the value is big (1000) for example then the animation will be very slow.

**Comment by matviishyn** [comment]
Same for me, lagging hardly, when I try to use native driver it stops playing

**Comment by doug-stritt** [comment]
I encountered the same problem when trying out react-native-progress. When I quit React Native Debugger the problem went away.

**Comment by RobertFOConnor** [comment]
Have you tried react-native-progress-wheel?

**Comment by markusl** [comment]
@deepslam @matviishyn @sepehr-laal are you still seeing the lag with the latest version? I just recently updated the example app and tested with an iPhone and could not see any issues.

**Comment by sepehr-laal** [comment]
@markusl I ended up using [Victory Native](https://github.com/FormidableLabs/victory-native). I also never had issues with perf on iOS. my issue was on Android Pixel XL.

**Comment by matviishyn** [comment]
> @deepslam @matviishyn @sepehr-laal are you still seeing the lag with the latest version? I just recently updated the example app and tested with an iPhone and could not see any issues.

As I mentioned above, it's laggy in dev mode and on some of low-end devices, I just ended up with `react-native-progress-wheel`, it doesn't require any dependencies

**Comment by avlonder** [comment]
I'm not sure, but I think you should be able to animate circle arcs of less than 180 degrees with useNativeDriver: true if you rotate the foreground circle using react native's transform style property and apply overflow: hidden to the background circle.
